frontend/src/pages/Requests/RequestItem/RequestButtons/RequestButtons.tsx

```diff
@@ -26,7 +27,7 @@ const RequestButtons: FC<IRequestProps> = ({
   <ButtonsContainer item xs={12} lg={4} rowSpacing={1}>
     <ActionButton
       variant="mediumContained"
-      onClick={() => approveRequest(id)}
+      onClick={() => approveRequest({ id })}
```

Comment:
```suggestion
      const handleApproveRequest = ({id}) => () => approveRequest({id})
      ...
      onClick={handleApproveRequest({ id })}
```

it this case, please, can you create container? Also all changes we can discuss at call.